java/core/src/main/java/co/worklytics/psoxy/rules/msft/PrebuiltSanitizerRules.java

```diff
@@ -26,7 +26,7 @@ public class PrebuiltSanitizerRules {
             .regex("^https://graph.microsoft.com/(.*)$")
             .build();
     static final Transform REDACT_ODATA_CONTEXT = Transform.Redact.builder()
-            .jsonPath("$.['@odata.context']")
+            .jsonPath("$..['@odata.context']")
```

Comment:
Change this, which affects to rest of MSFT yamls (because it is used across all MSFT-365). This is to redact all *context*, not only the one in the top of the JSON.